plugins/modules/swim_intent.py

```diff
@@ -182,10 +219,20 @@
             description: ActivateLowerImageVersion flag.
             type: bool
           device_upgrade_mode:
-            description: Swim Trigger Activation's deviceUpgradeMode.
+            description: It specifies the mode of upgrade to be applied to the devices having the following values - 'install', 'bundle', and 'currentlyExists'.
+              install - This mode instructs Cisco Catalyst Center to perform a clean installation of the new image on the target devices.
+                When this mode is selected, the existing image on the device is completely replaced with the new image during the upgrade process.
+                This ensures that the device runs only the new image version after the upgrade is completed.
+              bundle - This mode instructs Cisco Catalyst Center bundles the new image with the existing image on the device before initiating
+                the upgrade process.This mode allows for a more efficient upgrade process by preserving the existing image on the device while
+                adding the new image as an additional bundle.After the upgrade, the device can run either the existing image or the new bundled
+                image, depending on the configuration.
+              currentlyExists - This mode instructs Cisco Catalyst Center to checks if the target devices already have the desired image version
+                installed. If image already present on devices, no action is taken and upgrade process is skipped for those devices. This mode
+                is useful for avoiding unnecessary upgrades on devices that already have the correct image version installed, thereby saving time.
             type: str
-          distributeIfNeeded:
-            description: DistributeIfNeeded flag.
+          distribute_if_needed:
+            description: Set the distribute_if_needed flag while activating the swim image.
```

Comment:
`description: Enable the distribute_if_needed option when activating the SWIM image.`

Reply:
added.